scipy/stats/tests/test_multivariate.py

```diff
@@ -2479,3 +2481,107 @@ def test_random_state_property():
     for distfn, args in dists:
         check_random_state_property(distfn, args)
         check_pickling(distfn, args)
+
+class TestDirichletMultinomial:
+    def test_covariance_diagonal(self):
+        #Makes sure that the diagonal of the
+        #covariance matrix is the variance.
+        alpha = [1, 5, 3]
+        n = 3
+        c = dirichlet_multinomial.cov(alpha, n)
+        v = dirichlet_multinomial.var(alpha, n)
+
+        assert_array_equal(v, c.diagonal())
+
+    def test_variance(self):
+        alpha = [0.3, 2.1, 5.4]
+        n = 3
+        #Truncated to the 6th digit, so not
+        #exactly what we would expect from var
+        expected_v = [0.136162, 0.724381, 0.784293]
```

Comment:
I calculated them by hand. Does that still require a reference?

Reply:
Yes, please explain that in a comment (e.g. "Computed using a calculator from the definition in reference [1]"). Whoever provides the next round of review should not just trust that the values are correct, and they will either repeat the calculation themselves, or they will find another source of reference values. External implementations (e.g. in R) or published values are preferable because they are easier to verify, so this might end up being changed.

Reply:
Done.

Reply:
I looked for these comments in your latest commit, 903989824a62a2b4569328da5c770cd857619d85, but they were not there. Consequently, replaced the tests with numbers that I could verify. 
The PDF is tested against an implementation in R because if I were to test them using the same equations as are in the code, it's quite possible that I'd make the same mistake in each. 
I couldn't find an independent reference for the moments, so I tested them against the sample moments of a large sample from the distribution.